### ISSUE TITLE
Adding mt="20px" to Spinner crashes the application.

### ISSUE THREAD

**Issue by Shizuri** [GeekyAnts/NativeBase#4408]
**Bug description**
Adding a <Spinner> component with a string value for `mt` causes the application to crash with the error message:
```
Error while updating property 'marginTop' in shadow node of type: AndroidProgressBar

null

Unknown value: 20px
```

**To Reproduce**
Steps to reproduce the behaviour:

Add a 
`<Spinner mt="20px" />`
anywhere in the code.

Adding `<Spinner mt={20} />` does NOT cause the issue.

**Screenshots**
![image](https://user-images.githubusercontent.com/43163607/145801832-f1aca67e-d009-4be8-9a2d-b567e655cc67.png)

**Platform**

- NativeBase version 3.2.2
- Device: Android Studio Emulator
- OS: Windows 10
- React Native version 0.64.2
- Not using Expo
- node v14.17.6
- yarn 1.22.17

**Additional information**

1. Many times the notification is not shown before the application crashes.

**Comment by Viraj-10** [comment]
This Issue has been resolved in NativeBase v3.3.0 version.

**Comment by Shizuri** [comment]
@Viraj-10 Well that was fast. Well done! :)
